boards/Kconfig.v2

```diff
@@ -15,4 +16,23 @@ config BOARD_$(BOARD_FULL_STRING)
 	help
 	  Kconfig symbol identifying the board including full board identifier.
 
+config BOARD_QUALIFIERS
+	string
+	default "$(BOARD_QUALIFIERS_WITHOUT_SEPARATOR)"
+	help
+	  Contains the qualifiers of the board being used without the name of the board itself.
+
+	  For example, if building for ``nrf5340dk/nrf5340/cpuapp`` then this will contain the
+	  value ``nrf5340/cpuapp``.
+
+config BOARD_TARGET
+	string
+	default "$(BOARD)$(BOARD_IDENTIFIER)"
```

Comment:
This is not `BOARD_TARGET` according to 3.a here: https://github.com/zephyrproject-rtos/zephyr/issues/69905

It lacks the revision field.

Reply:
Then how can this be done in Kconfig given that BOARD_REVISION might not exist and if it does exist there is no @ symbol?

Reply:
Fixed